### PR TITLE
[WIP] ruby 2.4

### PR DESCRIPTION
**Pull request by mcfiredrill**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -276,12 +281,12 @@ DEPENDENCIES
...
   sass-rails (~> 5.0.0)
   sqlite3
-  therubyracer
+  therubyracer!
   uglifier (>= 1.0.3)
   unicorn
 
 RUBY VERSION
-   ruby 2.3.3p222
+   ruby 2.4.0p0
```
Comment (nikolas) [inline Gemfile.lock]: 0p0

Comment (mcfiredrill) [inline Gemfile.lock]: 🔫

**Comment by nikolas** [comment]
:3 https://www.ruby-lang.org/en/news/2017/03/22/ruby-2-4-1-released/

**Comment by mcfiredrill** [comment]
We are on 2.4.2 now

**Pull request closed without merging**